File: Python/BubbleSort.py
def BubbleSort(arr):
    n=len(arr)
    for i in range(n - 1):  # Passes
        for j in range(n - i - 1):  # Comparisons in each pass
            if arr[j] > arr[j + 1]:  # Swap if out of order
                arr[j], arr[j + 1] = arr[j + 1], arr[j]

arr = [5, 2, 9, 1, 5, 6]
BubbleSort(arr)
print("Sorted array:", arr)

File: C/BubbleSort.c
#include<stdio.h>
int BubbleSort(int arr[],int n){
	int i,j;
for(i=0;i<n;i++){
	for(j=0;j<n-i-1;j++){
		if(arr[j]>arr[j+1]){
			int temp=arr[j];
			arr[j]=arr[j+1];
			arr[j+1]=temp;
		}
	}
}
}
int main(){
	int arr[] = {5, 2, 9, 1, 5, 6};
    int n = sizeof(arr) / sizeof(arr[0]);

    BubbleSort(arr, n);
	int i;
    printf("Sorted array: ");
    for (i = 0; i < n; i++) {
        printf("%d ", arr[i]);
    }

    return 0;
}
